setup.sh

```diff
@@ -10,6 +10,7 @@ cd /vagrant
 # Install dependencies with Pipenv
 pipenv sync --dev
 
-# run our app. Nohup and "&" are used to let the setup script finish
-# while our app stays up. The app logs will be collected in nohup.out
-nohup pipenv run python manage.py runserver 0.0.0.0:8000 &
+# run our app. setsit, the parentheses and "&" are used to perform a "double
+# fork" so that out app stays up after the setup script finishes.
+# The app logs are redirected to the `runserver.log` file.
+(setsid pipenv run python manage.py runserver 0.0.0.0:8000 > runserver.log 2>&1 &)
```

Comment:
what made the app crash with what you used before? I never saw it happen.

Reply:
Always when we changed the code of the project the server was failed and we had to run the command:
 `vagrant provision`. 
Now it should be resolved as Barak Korren mentioned in the slack [here](https://redhat-beyond.slack.com/archives/C031MUZG0LE/p1647795038129999)

Reply:
Right, thanks. :)